packages/functional-tests/tests/syncV3/fxDesktopV3ForceAuth.spec.ts

```diff
@@ -115,7 +115,7 @@ test.describe('severity-1 #smoke', () => {
       const emailInputValue = await login.getEmailInput();
       expect(emailInputValue).toBe(email);
       const emailInput = await login.getEmailInputElement();
-      expect(emailInput.isDisabled());
+      expect(emailInput.isDisabled()).toBe(true);
```

Comment:
These were throwing an error w/:
```
Expect must have a corresponding matcher call  playwright/valid-expect
```

Reply:
But this should not have failed as this is the same exact thing mentioned in the [prefer-web-first-assertions](https://github.com/playwright-community/eslint-plugin-playwright/blob/main/docs/rules/prefer-web-first-assertions.md) to be used as correct code.